server/src/main/scala/net/wiringbits/validations/ValidateEmailIsRegistered.scala

```diff
@@ -0,0 +1,17 @@
+package net.wiringbits.validations
+
+import net.wiringbits.common.models.Email
+import net.wiringbits.repositories.UsersRepository
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object ValidateEmailIsRegistered {
```

Comment:
@caiseL don't we already have this?

Reply:
We have a similar validator to check if the email is available but in this case I don't think it'll work as it is right now https://github.com/wiringbits/scala-webapp-template/blob/master/server/src/main/scala/net/wiringbits/validations/ValidateEmailIsAvailable.scala